pagerduty.py

```diff
@@ -172,9 +173,9 @@ def lock_and_flush_queue(self):
 
     def enqueue(self, event):
         encoded_event = json.dumps(event)
-        process_id = os.getpid()
         time_seconds = int(time.time())
-        file_name = "%s/pd_%d_%d" % (self.queue_dir, time_seconds, process_id)
+        time_microseconds = str(datetime.datetime.utcnow().microsecond).zfill(7)
+        file_name = "%s/pd_%d%s" % (self.queue_dir, time_seconds, time_microseconds)
```

Comment:
I'd prefer to leave pid in there! Useful to reduce probability of file name collisions, though this is less likely with microseconds.